app/concepcts/screenings/use_cases/update.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Screenings
+  module UseCases
+    class Update
+      def initialize(id:, params:, repository: ScreeningRepository.new)
+        @id = id
+        @repository = repository
+        @params = params
+      end
+
+      def call
+        raise ActiveRecord::RecordNotFound if repository.find(id).present? == false
+
+        if find_movie(params) == false
+          raise ActiveRecord::RecordNotFound,
+                "Couldn't find move with id #{params[:movie_id]}"
+        end
+
+        repository.update(params, id)
+      end
+
+      private
+
+      attr_reader :params, :repository, :id
+
+      def find_movie(params)
```

Comment:
"find movie" means like this method will return movie, but this method will return information if this movie exists. Try to change naming to more accurate.